domain/snyk/authentication_functions.go

```diff
@@ -42,11 +40,6 @@ func GetActiveUser() (*ActiveUser, error) {
 		return nil, errors.New("no credentials found")
 	}
 	conf := c.Engine().GetConfiguration().Clone()
-	if c.AuthenticationMethod() == lsp.OAuthAuthentication {
```

Comment:
Question: Because we're enabling it by default we don't need to set these as they're configured already [here](https://github.com/snyk/go-application-framework/blob/cc0fb551c613340e4f9c5bca833e2de4caf48210/pkg/app/app.go#L185) right?

Reply:
I ran into some problems when not setting it explicitly, so I added it. In general, the FF should be removed from GAF.